index.css

```diff
@@ -514,4 +527,27 @@ html * {
   .section-title {
     width: 50%;
   }
+
+  .contact-background {
+    background-image: url("./assets/images/form-background-desktop.png");
+  }
+
+  .contact-background .contact-text {
+    max-width: 40%;
+    margin: 100px auto 20px auto;
+  }
+
+  #contact-form {
```

Comment:
- [ ] Kindly check the `font-size` of your contact section heading. It looks a bit bigger on the Figma design. 🧐️
- [ ] Kindly update your input elements to have crisp corners instead of rounded ones and some padding to them to match the template. 😎️
- [ ] I think your button is a bit too big when compared to the design. You can decrease the padding to make it smaller. 🤓️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/206831635-0f257a74-022f-4211-841a-5990836c48a2.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/206831634-d14487ab-4b71-4d9a-ad05-40c1450ba97f.png" width="300">|

Reply:
Thanks for the comment. I will fixed to accomplish the template